frank/plot.py

```diff
@@ -48,12 +48,12 @@ def plot_deprojection_effect(u, v, up, vp, vis, visp, ax0, ax1):
         Projected visibilities (either the real or imaginary component)
     visp : array
         Deprojected visibilities (either the real or imaginary component)
-    ax0 : Matplotlib `~.axes.Axes` class
-        Axis on which to plot effect of deprojection on the (u, v) coordinates
-    ax1 : Matplotlib `~.axes.Axes` class
-        Axis on which to plot effect of deprojection on the visibility amplitudes
+    axes : Matplotlib `~.axes.Axes` class
+        Axes on which to plot
```

Comment:
This should state that axes is a list[Axes, Axes]

Reply:
Good catch! Changed in 2658dfe